packages/jumpstarter-driver-flashers/oci_bundles/rcar_s4/add_kernel.sh

```
@@ -0,0 +1,72 @@
+	fixed
+	renesas_sdhi_internal_dmac
+	mmc_block
+	# R-Car S4 networking
+	gpio-rcar
+	r8a779f0-ether-serdes
+	marvell10g
+	rswitch
+)
+ODIR="overlay"
+
+cd "$(dirname "$(readlink -f "$0")")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)